knative-operator/deploy/resources/knativekafka/channel/2-channel-post-install.yaml

```diff
@@ -109,12 +109,13 @@ roleRef:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: storage-version-migration-kafka-channel-
+  generateName: storage-version-migration-eventing-kafka-channel-
```

Comment:
This name needs to match up with what we pass to the CSV.

https://github.com/openshift-knative/serverless-operator/blob/3f8441fda1b6092a49247f1aab328f3815df43c6/hack/generate/csv.sh#L77

/hold

Reply:
```
kafka_image "storage-version-migration-eventing-kafka-channel-__migrate" "${eventing_kafka}-storage-version-migration" 
```
should work